client/src/pages/HomePage.js

```diff
@@ -22,6 +22,9 @@ const HomePage = () => {
   const classes = useStyles();
   const [hasMore, setHasMore] = useState(true);
   const [mentionDatas, setMentionDatas] = useState(null);
+  const [switching, setSwitching] = useState(false);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [sort, setSort] = useState("date");
   const { dispatch, error, searchTerm, user } = useContext(UserContext);
 
   const loadMore = async (newPage) => {
```

Comment:
Can this be refactored to use the getMentions hook?

Reply:
Definitely !